python/ray/ml/train/integrations/sklearn/sklearn_trainer.py

```diff
@@ -112,26 +152,272 @@ def __init__(
         estimator: BaseEstimator,
         datasets: Dict[str, GenDataset],
         label_column: Optional[str] = None,
+        params: Optional[Dict[str, Any]] = None,
         scoring: Optional[ScoringType] = None,
         cv: Optional[CVType] = None,
+        parallelise_cv: Optional[bool] = None,
```

Comment:
I'm a big fan of British English but we should probably keep this consistent across the library
```suggestion
        parallelize_cv: Optional[bool] = None,
```